vars/maven.groovy

```diff
@@ -111,6 +111,20 @@ def mvnVersionsUpdateParentAndChildModules(MavenCommand mvnCmd, String newVersio
     mvnVersionsUpdateChildModules(mvnCmd, allowSnapshots)
 }
 
+def mvnGetVersionProperty(String property) {
+    mvnGetVersionProperty(new MavenCommand(this), property)
+}
+
+def mvnGetVersionProperty(MavenCommand mvnCmd, String property) {
+    def stdout = mvnCmd.clone()
+            .withOptions(['-q'])
+            .withProperty('expression', property)
+            .withProperty('forceStdout')
+            .returnOutput()
+            .run('help:evaluate')
+    return stdout ? stdout.trim() : ''
```

Comment:
PipelineMock returns null in case of an empty stdout, in pipeline it worked even without the check.